pyquaticus/envs/pyquaticus.py

```diff
@@ -78,7 +76,7 @@
     "tau": (
         1 / 10
     ),  # length of timestep (seconds) between state updates and for updating action input from demonstrator or rl
-    "max_time": 120.0,  # maximum time (seconds) per episode
+    "max_time": 240.0,  # maximum time (seconds) per episode
```

Comment:
Was this just for testing or would you like to change it permanently? Either is fine -- just checking.

Reply:
I was thinking permanently because the agents move much slower now (120 seconds was the default for the version with my old dynamics).